docs/configuring-playbook.md

```diff
@@ -54,3 +54,5 @@ When you're done with all the configuration you'd like to do, continue with [Ins
 - [Setting up Appservice IRC bridging](configuring-playbook-bridge-appservice-irc.md) (optional)
 
 - [Setting up Appservice Discord bridging](configuring-playbook-bridge-appservice-discord.md) (optional)
+
+- [Setting up Dimension (optional)](configuring-playbook-dimension.md) (optional)
```

Comment:
Besides this (which is okay), we can also link to the same documentation page from `installing.md`.

I guess that after `installing.md`'s:

```markdown
- or [create your first user account](registering-users.md)
```

line, we could add another one:

```markdown
- or [set up the Dimension Integrations Manager](configuring-playbook-dimension.md)
```

This is because Dimension cannot be configured by going through the normal flow for the first time (clone the playbook, read all  "Configuring" pages, adjust your configuration, run Install and you're done). Setting up Dimension requires that someone sets up the other Matrix services and then comes back to set up Dimension. So we should remind people of that after they've managed to create their first Matrix user.